aio/tests/e2e/src/app.e2e-spec.ts

```diff
@@ -5,7 +5,7 @@ describe('site App', () => {
   let page: SitePage;
 
   beforeEach(async () => {
-    await SitePage.setWindowWidth(1050);   // Make the window wide enough to show the SideNav side-by-side.
+    await SitePage.setWindowWidth(1200);   // Make the window wide enough to show the SideNav side-by-side (bigger than the app's showTopMenuWidth).
```

Comment:
Actually, this line is now above the 140 chars limit, so tslint barks.

Consider breaking it up:
```suggestion
    await SitePage.setWindowWidth(1200);   // Make the window wide enough to show the SideNav side-by-side
                                           // (bigger than the app's showTopMenuWidth).
```

...or putting it above the code:
```suggestion
    // Make the window wide enough to show the SideNav side-by-side (bigger than the app's showTopMenuWidth).
    await SitePage.setWindowWidth(1200);
```

Reply:
yeah sorry I missed that one :sweat: 

also I needed to put it in two lines to make the linting pass
(since it was already in two lines I also added "component" to make the comment clearer :slightly_smiling_face: )

Reply:
By the way, are there any plans to pass to eslint? :slightly_smiling_face:

Reply:
> By the way, are there any plans to pass to eslint? 🙂

No concrete plans atm. Did you just volunteer? 😛

Reply:
yeah, I mean if you want to I would not mind giving it a shot! :smiley:

Reply:
Let me discuss this with the team and I'll let you know 🚀

Reply:
We discussed this and we are on-board. If you want to give it a go, you are mostly welcome to do so :smiley:

Reply:
Awesome! :slightly_smiling_face: 
I am a bit snowed under so I think I will start to look at this tomorrow or weekend, let me know if you want me to create an issue so we can keep track of things there :slightly_smiling_face:

Reply:
Whenever works for you 🚀

No need to create an issue, but you find it helpful to track work, feel free to create one.
Also feel free to hit me up if you get blocked on anything or have any questions.

Reply:
Awesome, ok I will just create an issue if I find the need for one :slightly_smiling_face: 

I won't hesitate in contacting you then! thanks a lot!!! :slightly_smiling_face: